.circleci/config.yml

```diff
@@ -652,3 +294,20 @@ workflows:
   build-mas-x64:
     jobs:
       - electron-mas-x64
+
+  nightly-release-test:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - master
+                - 2-0-x
+                - 1-8-x
+                - 1-7-x
```

Comment:
wondering about this branch list -- we're doing nightlies for each active branch?

Reply:
Yup, that's the plan